core/src/main/scala/caliban/relay/Base64Cursor.scala

```diff
@@ -0,0 +1,40 @@
+package caliban.relay
+
+import caliban.schema.Schema
+import caliban.Value
+import scala.util.Try
+
+/**
+ * A cursor implementation that models an index/offset as an
+ * opaque base64 cursor.
+ */
+case class Base64Cursor(value: Int)
+object Base64Cursor {
```

Comment:
Do you mind adding a line break, just to match the style of the rest of the code 😄

Reply:
👍